src/Microsoft.DotNet.Darc/src/Darc/Operations/VirtualMonoRepo/VmrOperationBase.cs

```diff
@@ -56,6 +56,11 @@ public override async Task<int> ExecuteAsync()
                 .ToImmutableArray();
         }
 
+        IVmrInfo vmrInfo = Provider.GetRequiredService<IVmrInfo>();
+
+        vmrInfo.ReadmeTemplatePath = _options.ReadMeTemplate;
```

Comment:
Can we not put this in the `VmrInfo` but have as parameters instead? I think it would be better API to have `UpdateReadme(string templatePath)`

Reply:
Same for TPN